hack/make-rules/test.sh

```diff
@@ -202,10 +203,18 @@ for arg; do
   fi
 done
 if [[ ${#testcases[@]} -eq 0 ]]; then
-  testcases=($(kube::test::find_dirs))
+  while IFS='' read -r line; do testcases+=("$line"); done < <(kube::test::find_dirs)
 fi
 set -- "${testcases[@]+${testcases[@]}}"
 
+if [[ -n "${KUBE_RACE}" ]] ; then
+  goflags+=("${KUBE_RACE}")
+fi
+
+if [[ -n "${KUBE_TIMEOUT}" ]] ; then
+  while IFS='' read -r line; do goflags+=("$line"); done < <(echo "${KUBE_TIMEOUT}" | tr ' ' '\n')
```

Comment:
why are we doing this?

Reply:
This is for adding KUBE_RACE and KUBE_TIMEOUT to goflags array.

KUBE_TIMEOUT is "--timeout 120s" as default.
If it is changed to `goflags+=("${KUBE_TIMEOUT}")` like KUBE_RACE, "-timeout 120s" is recognized as one parameter, and `go test` cannot run.
Therefore, KUBE_TIMEOUT(= -timeout 120s) is divided into "--timeout" and "120s" on this code.

Reply:
can we update KUBE_TIMEOUT to --timeout=120s instead?

Reply:
this change feels suspect, otherwise LGTM.

this is also a pretty awkward way to split on spaces, bash does that already with just unquoting the env. fixing KUBE_TIMEOUT is probably a better answer

Reply:
Thanks for the advice.
I confirmed that `go test` with `-timeout=120s` could be run. so, I will change that.
`cmd/kubeadm.sh` and `test-integration.sh` have KUBE_TIMEOUT, so I will change, too.

Reply:
@BenTheElder could you please review?